safety_gym/envs/maze.py

```diff
@@ -62,6 +62,8 @@ def maze2():
 
 
 wall = maze1()
+t = Translation(wall)
+t.start_search("2d_nav")
```

Comment:
because we have changed to safety gym, the name may be changed to `safety-gym-maze-*`.

Reply:
I called it safety-gym-maze-1 - let me know if I should change it.